assets/js/components/Collection/Collection.jsx

```diff
@@ -1,18 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
 import PropTypes, { shape } from "prop-types";
+import CollectionImageModal from "../../screens/Collection/CollectionImageModal";
+
+const Collection = ({ collection }) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const {
+    adminEmail,
+    description,
+    representativeImage,
+    findingAidUrl,
+    keywords = [],
+  } = collection;
+
+  const onCloseModal = () => {
+    setIsModalOpen(false);
+  };
 
-const Collection = ({
-  adminEmail,
-  description,
-  findingAidUrl,
-  keywords = []
-}) => {
   return (
     <div data-testid="collection">
       <div className="columns">
         <div className="column is-one-quarter-desktop is-half-tablet">
           <figure className="image is-square">
-            <img src="https://bulma.io/images/placeholders/480x480.png" />
+            <img
+              src={
+                representativeImage != null
+                  ? representativeImage + "/square/500,500/0/default.jpg"
+                  : "/images/480x480.png"
+              }
+            />
+            <button
```

Comment:
Can we move this `button` out of the `figure` element?  Figures typically only contain an `img` elements.   And instead of a regular looking button, let's make it a "text" button that's small.